doc/game.md

```diff
@@ -150,6 +150,19 @@ built upon two methods:
 
 The `GameLoop` is used by all of Flame's `Game` implementations.
 
+# Pause/Resuming game execution
+
+A Flame `Game` can be paused and resumed in two ways:
+
+ - With the use of the `pauseEngine` and `resumeEngine` methods.
+ - By changing the `paused` attribute.
+
+When pausing a Flame `Game`, the `GameLoop` is effectively paused, meaning that no updates and new
+renders will happen until it is resumed.
+
+To pause/resume the update of custom components, custom logic needs to be implemented on each
```

Comment:
Why does it need custom logic if the main game loop is paused?

Reply:
I meant, like if you want to pause a specific component, not all of them. Any suggestion on to make this more clear?

Reply:
Aah then I get it, but do we need to talk about that on here, since it is in a section about the game?
If you want to write something about it anyways, maybe you could formulate it something like this:
"If you want to pause a specific component when the full game isn't paused, you'll have to implement custom logic for that."

Reply:
I was on the fence about quoting that or, I will remove then 😉